City-of-Helsinki/helsinki-paatokset
# PP-284: HDBT syncing

Make site compatible with the most recent version of HDBT. 
- A lot of obsolete style changes removed, letting HDBT override them.
- Break policymaker side menu into 2 parts (own block for mobile/desktop)
- Use variables from preprocess hooks to determine how page layout is rendered

To test:
- Chekout branch. Run `make new` (take a copy beforehand with `make drush-create-dump`)

Things to test for: 
- Make a landing page 'Etusivu'. Try adding all kinds of paragraph types to it - check that they look OK in both desktop and mobile. Try adding the hero block of type 'search form' and see that it aligns correctly with the added paragraphs.
- Try adding a page to the main menu. This should cause no errors, and the link should be displayed correctly. Try adding a sublink for that menu item. Again, there should be no errors, and the main navigation should look correct (with an angle-down icon). Check that the main menu works in mobile.
- Get some integration data: run `drush ap:fs; drush mim ahjo_meetings:latest; drush mim ahjo_cases:latest; drush mim ahjo_decisions:latest; ahjo_decisionmakers:all; ahjo_trustees:all`. Check some of the case / policymaker / trustee pages and their subpages and see if you can spot anything wonky. On policymaker / trustee pags, check that the mobile navigation works as intended. Also check that the main menu is not shown in the sidebar on policymaker / policymaker subpages.
- Try out any custom-made blocks you can think of - pay attention to how they align in the page. Note that **search applications** will be fixed in this regard in an upcoming ticket, disregard these for now.
